gui/aju_flask/aju_app/__init__.py

```diff
@@ -20,3 +37,48 @@ def create_app(config_name):
     app.register_blueprint(main_blueprint)
 
     return app
+
+
+@socketio.on('connect')
+def send_kafka_data():
+    global thread
+    with thread_lock:
+        if thread is None:
+            thread = socketio.start_background_task(target=background_send_kafka_data_thread)
+
+
+def background_send_kafka_data_thread():
+    with open(OUTPUT_DATA_FILE, 'r') as f:
+        while True:
+            time.sleep(0.1)
+            line = f.readline()
+            if line:
+                line_list = line.strip().lstrip('(').rstrip(')').split(',')
+                print(time.strftime("%Y-%m-%d %H:%M:%S", time.localtime()) + "  send: " + str(line_list))
+                socketio.emit('result_figure_data', {'data': line_list})
+            else:
+                f.seek(0)
+
+
+def background_send_kafka_data_thread_real():
+    KAFKA_HOME_PATH = '/Users/chaoqi/Programs/kafka_2.12-2.6.0'

```

Comment:
Because this is temporarily are constant variables. To just make the code more readable, I set all the constant variables uppercases, and the other variables as lowercases.